apps/mobile/src/screens/adoptions/components/adoptions-list.tsx

```diff
@@ -0,0 +1,38 @@
+import { AdoptionType } from '@animavita/types';
+import { Box, Heading, View } from 'native-base';
+import { FlatList } from 'react-native';
+
+import { AdoptionCard } from './adoption-card';
+
+type Adoption = AdoptionType & {
+  id: string;
+};
+
+type AdoptionsListProps = {
+  adoptions: Adoption[];
+};
+
+export const AdoptionsList = ({ adoptions }: AdoptionsListProps) => {
+  if (adoptions.length <= 0) return <EmptyList />;
+
+  return (
+    <FlatList
+      renderItem={({ item }) => <AdoptionCard {...item} />}
+      data={adoptions}
+      keyExtractor={({ id }) => id}
```

Comment:
Can you build a more meaningful string to prevent it from conflicting with other lists? Something like: 

`keyExtractor={({ id }) => `my-adoption-item-${id}`}`